auditbeat/Jenkinsfile.yml

```diff
@@ -13,6 +13,11 @@ when:
     tags: true                 ## for all the tags
 platform: "linux && ubuntu-18" ## default label for all the stages
 stages:
+    Lint:
+        make: |
+          make -C auditbeat check;
+          make -C auditbeat update;
+          make check-no-changes;
```

Comment:
IIUC, `check` calls `check-no-changes` but it's called explicitly since the scope it's within the whole workspace rather than the `auditbeat` folder?

Reply:
the `check` target on beats does not call `check-no-changes` target, the `check` and `update` targets can make changes on files so we have to run `check-no-changes` to ensure everything is OK.